test.py

```diff
@@ -60,6 +60,41 @@ def test_get_experiment_info(self):
                 test_info["ExperimentClass"].model_dump()
             )
 
+    def test_get_experiment_queue(self):
+        test_queues = tuple({
+            '1': {'pipeline': 'main',
+                'expid': {
+                    'log_level': 30,
+                    'class_name': None,
+                    'arguments': {'user': 'QuIQCL', 'time': 1.0, 'save': False, 'color': 'r'},
+                    'file': 'DIRECTORY-PATH'
+                },
+                'priority': 1,
+                'due_date': None,
+                'flush': False,
+                'status': status,
+                'repo_msg': None
+            }
+        } for status in ['pending', 'preparing', 'running', 'run_done', 'analyzing', 'deleting'])
```

Comment:
I think the indentation looks not legible.
How about this?

```python
test_queues = tuple({
  '1': {
    'pipeline': 'main',
    'expid': {
      'log_level': 30,
      'class_name': None,
      'arguments': {'user': 'QuIQCL', 'time': 1.0, 'save': False, 'color': 'r'},
      'file': 'DIRECTORY-PATH'
    },
    'priority': 1,
    'due_date': None,
    'flush': False,
    'status': status,
    'repo_msg': None
  }} for status in ['pending', 'preparing', 'running', 'run_done', 'analyzing', 'deleting']
)
```

Reply:
If you cannot make it in a single line, do not use comprehension.
The longest acceptable form:
```python
generator = (
    long_expression
    for variable in iterable
    if condition
)
```
If the expression does not fit in a single line, please consider using a `for` statement, etc.

Reply:
In addition, please use double quotes (`"`) for consistency.